xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-filters/xwiki-platform-notifications-filters-api/src/test/java/org/xwiki/notifications/filters/internal/scope/ScopeNotificationFilterTest.java

```diff
@@ -268,7 +267,6 @@ void withTopLevelInclusiveFilters() throws Exception
         DocumentReference documentReference = new DocumentReference("wikiA", "SpaceM", "DocumentN");
         NotificationFilterPreference prefζ = mockNotificationFilterPreference("wikiA:SpaceM.DocumentN",
                 documentReference, NotificationFilterType.INCLUSIVE, null);
-        when(prefζ.getProviderHint()).thenReturn("userProfile");
         when(prefζ.getStartingDate()).thenReturn(new Date(99000));
```

Comment:
Not something introduced by this pull request by that's a pretty surprising character to use in a variable name.